src/App.tsx

```diff
@@ -1,61 +1,124 @@
-import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
```

Comment:
Commit and push all files
![2023-09-21_15h38_33](https://github.com/mate-academy/react_add-todo-form/assets/61904995/f5f431ad-6d97-400f-ae67-ee2ffb65804e)